feconf.py

```diff
@@ -708,6 +708,7 @@ def get_empty_ratings():
     u'23': 'rating_test.yaml',
     u'24': 'learner_flow_test.yaml',
     u'25': 'exploration_player_test.yaml',
+    u'26': 'android_interactions',
```

Comment:
To load the exploration from the admin activities tab similar to other dummy explorations

Reply:
Just a note -- probably OK to have this here, but you probably won't end up using this route when calling the custom handler that you're writing.

Reply:
but [exp_services.load_demo](https://github.com/oppia/oppia/blob/develop/core/domain/exp_services.py#L1500) needs exploration id right?

Reply:
Yes, but you may be able to specify that without referencing this directly (since this list is more for the "reload all" button in the admin panel).

Reply:
It's a bit nice for manual debugging of the exploration, though.